starknet_py/tests/e2e/block_test.py

```diff
@@ -28,3 +28,41 @@ async def test_latest_block(account, map_compiled_contract):
 
     blk = await account.client.get_block(block_number="latest")
     assert blk.block_hash
+
+
+@pytest.mark.parametrize("block_number", ("pending", "latest"))
+@pytest.mark.asyncio
+async def test_block_with_tx_hashes(
+    full_node_account,
+    block_number,
+    map_contract_declare,  # pylint: disable=unused-argument
+):
+    blk = await full_node_account.client.get_block_with_tx_hashes(
+        block_number=block_number
+    )
+
+    assert isinstance(blk.transactions, list)
```

Comment:
Let's check that the block with tx hashes actually does have hashes of transactions